src/implementations/utils/wallet.ts

```diff
@@ -56,17 +56,25 @@ export const getWalletOrSigner = async ({
     signale.info("Wallet successfully decrypted");
     return wallet.connect(provider);
   } else if (isAwsKmsSignerOption(options)) {
-    const { secretAccessKey } = await inquirer.prompt({
-      type: "password",
-      name: "secretAccessKey",
-      message: "Secret access key",
-    });
-    const kmsCredentials = {
-      accessKeyId: options.accessKeyId, // credentials for your IAM user with KMS access
-      secretAccessKey, // credentials for your IAM user with KMS access
-      region: options.region,
-      keyId: options.kmsKeyId,
-    };
+    let kmsCredentials;
+    if (options.useEnv) {
```

Comment:
the branch doesn't look necessary, except to retrieve the secret.

We can build the kmsCredentials Once whether the values are set. If the access key is not set it will be undefined

I would suggest we drop the inquirer and add the secret access directly in the cli. If people want to use a more secure options there are already possibilities from the lib itself.

Reply:
removed branch, strangely my editor is now showing this error but i tested using npm run dev and everything still works as expected.  

<img width="1136" alt="Screenshot 2021-08-07 at 3 01 36 PM" src="https://user-images.githubusercontent.com/31716292/128802532-c3f6a6cf-3fd8-4c04-aec6-e7c558664503.png">